### ISSUE TITLE
TBoxCSV type in the data uploader requires owl filename to be in the IRI

### ISSUE THREAD

**Issue by kok-foong** [cambridge-cares/TheWorldAvatar#856]
I believe the agreed prefixes for our ontologies should be in the form of https://www.theworldavatar.com/kb/ontosomething/, ending with a slash, not # or the owl filename. However, the data uploader does not seem to like that. If the IRI of the TBox does not contain the owl filename, it will throw an exception.

e.g. in usually the first line of our csv files,
OntoDispersion,TBox,https://www.theworldavatar.com/kg/ontosomething/,https://www.w3.org/2007/05/powder-s#hasIRI
will fail, and 
OntoDispersion,TBox,https://www.theworldavatar.com/kg/ontosomething/OntoSomething.owl,https://www.w3.org/2007/05/powder-s#hasIRI
will pass.

I think using `public void generateTBox(String csvFileNamePlusPath)` instead of `public void generateTBox(String csvFileNamePlusPath, String outputFilePlusPath)` might solve this.

**Comment by gpeb2** [comment]
I don't think this error is coming from the data uploader as it just passes file names to the `TBoxGeneration` object. And that other version of the `generateTBox` method would only add the generation of the LaTeX file.
I think it is the `manager.getOntologyFormat(ontology)` argument in [this](https://github.com/cambridge-cares/TheWorldAvatar/blob/4e41776451b2c89d2e1589710d49f4ac2ed50734/JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/TBoxManagement.java#L1466) line that is the issue. Setting that to `.owl`, or even better taking the extension from the `owlFilePath` argument, will probably fix the issue.
